src/declarations/stencil-public-compiler.ts

```diff
@@ -2233,8 +2234,15 @@ export interface OutputTargetHydrate extends OutputTargetBase {
 export interface OutputTargetCustom extends OutputTargetBase {
   type: 'custom';
   name: string;
+  /**
+   * Indicate in wich mode the output target has to be executed.
+   * By default if nothing is specified it run for both.
+   * - `"run"`: Executed in dev mode on every change in your code
+   * - `"build"`: Executed only on build
+   */
+  task?: 'run' | 'build';
```

Comment:
I don't see any docs for custom output targets so having the additional TS doc comment seems sufficient at this point.